### DISCUSSION TITLE
Problem installing Klipper with Kiauh Tutorial Guilouz

### DISCUSSION THREAD

**Discussion by BoB-DeS-BoiS** [Guilouz/Klipper-Flsun-Speeder-Pad#186] [Q&A]
Good evening everyone, I followed Guilouz's tutorial, everything went very well but when installing Klipper using Kiauh I found myself with an error message during the installation of Python 3.9.19 virtual environment I attach the photo, I spent 2 days trying to understand but I can't find... if anyone has a magic command! thank you in advance.
![Capture d’écran 2024-09-03 225415](https://github.com/user-attachments/assets/810eb8a3-d510-4dca-adea-62a6881dca36)

**Comment by Hydra20vt** [comment]
I just encountered the same problem yesterday. Virtualenv creation failure due to setuptools >= 71.0.0 
After a long session of insults, intensive search and nosense commands, I finally came up with a solution:

Type:
"pip install setuptools==70.3.0" 
and relauch the Klipper installation from Kiauh

Hope it helps

**Reply by se-SteveThePirate** [reply]
That fixed it for me, cheers.

**Comment by BoB-DeS-BoiS** [comment]
thank you very much I was finally able to do the updates :)
